Bricks/util/test.cc

```diff
@@ -517,6 +518,44 @@ TEST(Util, LazyInstantiation) {
   EXPECT_EQ("400:7", bar_y_q.InstantiateAsUniquePtrWithExtraParameter(400)->AsString());
 }
 
+TEST(Util, Iterator) {
+  typedef std::map<int, std::string> map1_t;
+  map1_t map1;
+  auto accessor1 = current::GenericMapAccessor<map1_t>(map1);
+  EXPECT_TRUE(accessor1.Empty());
+  EXPECT_EQ(0u, accessor1.Size());
+  EXPECT_EQ(accessor1.begin(), accessor1.end());
```

Comment:
This probably best as `EXPECT_TRUE(accessor1.begin() == accessor1.end())`. What the `EXCEPT_EQ` would print as `Actual:` and `Expected:` would be unreadable anyway.